README.md

```diff
@@ -8,10 +8,28 @@
 **Lawnicons** is an icon pack developed by the Lawnchair team and supported by our community.
 Originally an addon for Lawnchair 12 Alpha 5 and above to implement themed icons, it can now be used on many launchers.
 
-Lawnicons is best used [on Lawnchair 12.1 or Lawnchair 14 Beta 2](https://github.com/LawnchairLauncher/lawnchair/releases). You can enable themed icons on Lawnchair by going to `Home Settings → General → Icon Style → Themed Icons → "Home screen" or "Home screen and App Drawer"`.
+Lawnicons is best used [on Lawnchair 12.1 or later](https://github.com/LawnchairLauncher/lawnchair/releases). You can enable themed icons on Lawnchair by going to `Home Settings → General → Icon Style → Themed Icons → "Home screen" or "Home screen and App Drawer"`.
```

Comment:
Then it's better this way: "on Lawnchair 14 Beta 2 or later".

In version 12.1, some icons were installed incorrectly and this was fixed in Beta 14.2.

Reply:
Would it be better to say `the latest version of Lawnchair`?There could always be new problems discovered with certain versions of Lawnchair relating to Lawnicons, so it might just be better to encourage people to use the latest version.